pandas/io/parquet.py

```diff
@@ -260,8 +260,8 @@ def read_parquet(path, engine='auto', columns=None, **kwargs):
 
     Parameters
     ----------
-    path : string
-        File path
+    path : str
```

Comment:
Should this be path-like (anything implementing __fspath__)? I suppose that's up to the engine. I don't think it's correct to say that only strings are supported here.

Reply:
I *believe* pyarrow accepts pathlike (cc @wesm)  and fastparquet (cc @martindurant)

Reply:
yes